AlphaWallet/TokenScriptClient/Models/FunctionOrigin.swift

```diff
@@ -80,11 +80,13 @@ struct FunctionOrigin {
     private let originContractOrRecipientAddress: AlphaWallet.Address
     private let attributeId: AttributeId
     private let functionType: FunctionType
+    private let bitmask: BigUInt
```

Comment:
I think we should either:

A) make the `bitmask` property optional, and we try to read it before applying it, either fallback to `0xF...F`, or even better skip the bitmask operation entirely
B) In the 3 `.init()` fallback to `0xF..F` (then we can't skip the bitmask operation like in A above)

The 2 bitmask value in `XMLHandler.createFunctionOriginFrom()` shouldn't be `BigUInt(0)`, they should be nil then A or B above take care of it.